File: star3/circle_area.c
#include <math.h> // 円周率 M_PI が定義されています
#include <stdio.h>

int main(int argc, char const *argv[]) {
  int radius = 10;                     // 円の半径
  double area = pow(radius, 2) * M_PI; // 円の面積
                                       // pow関数で、2乗を求めています。

  printf("円周率 = %f\n", M_PI);   // 特に書式を指定しなかった場合
  printf("円周率 = %.4f\n", M_PI); // 小数点以下４桁表示
  printf("円周率 = %.15f\n", M_PI); // 小数点以下１５桁表示
  printf("半径 %d の円の面積は、%.15f です。\n", radius, area);

  return 0;
}

File: star3/circle_area.rb
radius = 10 # 円の半径
# 冪乗演算子 ** で2乗を求めています。
# Math::PI は Mathモジュール内で、円周率PIとして定義されている定数です。
area   = radius ** 2 * Math::PI # 円の面積

printf "円周率 = %f\n", Math::PI # 特に書式を指定しなかった場合
printf "円周率 = %.4f\n", Math::PI # 小数点以下４桁表示
printf "円周率 = %.15f\n", Math::PI # 小数点以下１５桁表示
printf "半径 %d の円の面積は、%.15f です。\n", radius, area
